src/main/java/org/mamute/providers/SolrServerProvider.java

```diff
@@ -50,7 +50,9 @@ public void create() {
 			LOGGER.info("Starting embedded Solr");
 			String home = solrHome;
 			if (isEmpty(home)) {
-				home = env.getResource("/solr").getPath();
+				String locale = env.get("locale", "en");
```

Comment:
I guess we could just put 'en' as default locale at mamute.properties so we don't need to spread this around the code. Right @csokol ?